### PR TITLE
driftignore prevent resource fetch

### PR DESCRIPTION
**Pull request by moadibfr**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | yes
| 🚀 New feature?   | yes
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #456 
| ❓ Documentation  | yes 
## Description

driftignore prevent resource fetch

### CODE REVIEW

File: pkg/resource/supplier.go [hunk 1]
```diff
@@ -1,10 +1,15 @@
 package resource
 
-// Resource Supplier supply the list of resource.Resource, its the front to retrieve remote resources
+// Supplier Resource Supplier supply the list of resource.Resource, its the front to retrieve remote resources
```
Comment (lotoussa) [inline pkg/resource/supplier.go:3]: ?

**Review by lotoussa — APPROVED** [review]
it seems to me that everything is fine

File: pkg/resource/supplier.go (hunk 1 above)
Comment (moadibfr) [reply pkg/resource/supplier.go:3]: I'll revert this, my linter does not like when comment don't start with the object name as it does not respect godoc... https://blog.golang.org/godoc

**Comment by wbeuil** [comment]
Focus is on the `deep mode` #619, let's put it in draft for now

**Comment by moadibfr** [comment]
need too much rework we should consider writing differently for the new arch that came for deepmode

**Pull request closed without merging**